app/api/wallet/generate/route.ts

```diff
@@ -5,14 +5,6 @@ import { PrismaClient } from "@/generated/prisma";
 export async function POST() {
   try {
     const prisma = new PrismaClient();
```

Comment:
Can remove this as it's no longer referenced anywhere else in this file. (Same with the import on line 3)